galasa-simbank-tests/dev.galasa.simbank.tests/pom-example.xml

```diff
@@ -23,7 +23,7 @@
 			<dependency>
 				<groupId>dev.galasa</groupId>
 				<artifactId>galasa-bom</artifactId>
-				<version>0.25.0</version>
```

Comment:
blimey. that's old. I think we need a set-version.sh script to bump this too. I'll add that scope to my bumpp-up story.